src/test/city-card.test.jsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect.js';
+import { Provider, useDispatch, useSelector } from 'react-redux';
+import store from '../redux/store';
+import CityCard from '../components/city-card.component';
+import axios from 'axios';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { BrowserRouter, MemoryRouter } from 'react-router-dom';
+
+jest.mock('axios', () => ({
+  post: jest.fn(),
+  get: jest.fn(),
+}));
+
+const reactRedux = { useDispatch, useSelector };
+
+describe('Render the detail info of city', () => {
+  
+  const useSelectorMock = jest.spyOn(reactRedux, 'useSelector')
+  const useDispatchMock = jest.spyOn(reactRedux, 'useDispatch')
+
+  beforeEach(() => {
+    useDispatchMock.mockClear();
+    useSelectorMock.mockClear();
+  })
+  afterEach(() => {
+      useDispatchMock.mockClear();
+      useSelectorMock.mockClear();
+  })
+
+  const city = {
+    id: 5,
+    name: 'one',
+    aqi: 10,
+    time: {
+      s: '2022-21-12 17:00'
+    },
+  }
+  
```

Comment:
- Good job adding tests to your project, but some of your tests are failing, please ensure you run all tests and see that they all pass. Please fix this 👍 👍 

Your tests:
![rgawrg](https://user-images.githubusercontent.com/73607512/225969318-a5ebb00f-5953-4ea5-91b2-ce4a2283d436.PNG)

Reply:
Tests all pass. Congratulations!!! 🎆 👏🏾 

![image](https://user-images.githubusercontent.com/38088818/226093322-af90d6d9-c0e5-40d5-bbd6-d065be5d2875.png)